src/ucp/core/ucp_ep.c

```diff
@@ -1067,6 +1067,12 @@ static void ucp_ep_config_set_rndv_thresh(ucp_worker_t *worker,
     } else {
         rndv_thresh     = context->config.ext.rndv_thresh;
         rndv_nbr_thresh = context->config.ext.rndv_thresh;
+
+        /* adjust max_short if rndv_thresh is set externally */
+        if (config->tag.eager.max_short > 0) {
+            config->tag.eager.max_short = ucs_min(config->tag.eager.max_short,
```

Comment:
what is about zcopy_thresh? Would not be bcopy selected up to zcopy_thres, even though rndv is 0?
do you have some test for 0 size rndv?

Reply:
zcopy_thresh (and as a result max_bcopy) is updated during runtime in tag send procedure
yes, i'm running ucx_perftest with UCX_RNDV_THRESH=0 for message < 64 bytes and it fails due to RNDV GET Scheme